ember-cli-build.js

```diff
@@ -2,9 +2,6 @@
 
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
-const environment = process.env.EMBER_ENV;
```

Comment:
these were added in one commit, and removed in this commit.

that isnt suitable for pushing to a PR for mentors to have to find and complain about.